library/build.gradle

```diff
@@ -65,7 +67,8 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:support-v4:25.3.1'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
what is the legacy support for? it would be nice to provide a comment for future devs to understand the purpose

Reply:
Not sure what to comment. The library has just been replaced by AndroidX version as described in https://developer.android.com/jetpack/androidx/migrate. The whole commit is about migrating to AndroidX -- this line in particular.

Reply:
in the wandera app we use the support v13 lib, so I was confused about the difference